literatur.bib

```diff
@@ -2675,6 +2675,13 @@ @Proceedings{celorrio-et-al-2011
   year =         "2011"
 }
 
+@InProceedings{celorrio-et-al-icaps2013wspal,
+  author =       "Sergio J. Celorrio and Patrick Haslum and Sylvie Thiebaux",
```

Comment:
Add accent to Sylvie's last name (see other entries).

Reply:
There is a also a typo in Patrik Haslum's name.

Reply:
(Also for at least one more entry -- I didn't look at the whole diff, only at the parts Jendrik answered to.)